runtime/doc/deprecated.txt

```diff
@@ -48,8 +48,10 @@ Modifiers ~
 *:map-special*		<> notation is always enabled. |cpo-<|
 
 Options ~
+*'gd'*
+*'gdefault'*		Enables the |:substitute| flag 'g' by default.
```

Comment:
Should not be a tag  unless the other entry was moved here entirely. In the meantime it's ok just to mention it here.